### PR TITLE
test(web): add e2e test example to visualizer

### PR DESCRIPTION
**Pull request by ZTongci**
# Overview
add e2e test example to visualizer

## What I've done
create session once and use this session to login and testing.
add pom structure for e2e test.
add e2e test example to visualizer that create layer and add circle into earth.

## What I haven't done
I don't change old e2e test structure. And didn't use auth0 login function.

## Summary by CodeRabbit

- **New Features**
	- Added a new test script for creating a project and adding a layer, including visual validation through screenshots.
	- Implemented a structured interface for interacting with map visualization components in end-to-end testing.
	- Introduced new environment variables to support end-to-end testing configurations.

- **Bug Fixes**
	- Disabled an inactive test case for logging into the dashboard to prevent misleading results.

- **Documentation**
	- Enhanced module exports for better organization and accessibility of components related to project and map functionalities.

- **Chores**
	- Updated `.gitignore` to exclude sensitive files and directories from version control.
	- Improved Playwright configuration for enhanced test execution and debugging.

### CODE REVIEW

File: web/e2e/dashboard.spec.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
-  await reearth.initUser();
-  await reearth.goto(`/dashboard/${reearth.teamId}`);
+// test("dasboard can be logged in", async ({ page, reearth }) => {
+//   await reearth.initUser();
+//   await reearth.goto(`/dashboard/${reearth.teamId}`);
 
-  await expect(page.getByText(`${reearth.userName}'s workspace`)).toBeVisible();
-});
+//   await expect(page.getByText(`${reearth.userName}'s workspace`)).toBeVisible();
+// });
```
Comment (airslice) [inline web/e2e/dashboard.spec.ts:8]: Why should these been commented out?

Comment (ZTongci) [reply web/e2e/dashboard.spec.ts:9]: it is no use file but include old login test. So save it temporary, I add no use comment. Please check.
https://github.com/reearth/reearth-visualizer/pull/1140/commits/3261d87b82731dd830bbecad32c6194daed624bf

File: web/e2e/example.spec.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    )
+    .toMatchSnapshot({ name: "expect.png", maxDiffPixels: 10 });
+
+  //delete created project
+  await dashboardPage.goto();
+  await dashboardPage.intoProjectSettingByName(testTitle);
+  const generalSettingPage = new pom.visualizer.projectSetting.generalPage(
+    page
+  );
+  await generalSettingPage.deleteProject();
```
[the remark this answers is not in the document]
Comment (ZTongci) [reply web/e2e/example.spec.ts]: I change the title of "_", others can not chaange.
https://github.com/reearth/reearth-visualizer/pull/1140/commits/1e58a33c4a9998df79efd767c1fbe73f98575954

File: web/e2e/README.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+```
+
+To set required .env for frontend. Recording following variable.
+
+```bash
+# Local backend with Auth0 OSS tenant
+REEARTH_WEB_E2E_BASEURL=http://localhost:3000
+REEARTH_WEB_E2E_ACCOUNT // your visualizer accounnt
+REEARTH_WEB_E2E_ACCOUNT_PASSWORD // your visualizer password
+```
```
Comment (mulengawilfred) [inline web/e2e/README.md:58]: Sorry, we might need to add these to `.env.example` file which acts as a template for what variables are required to run visualizer.

Comment (ZTongci) [reply web/e2e/README.md:58]: Thank you for your advice. I already add these three variables to `.env.example`. Please check.
https://github.com/reearth/reearth-visualizer/pull/1140/commits/e9160e5ece3adad59c381582a337f24662533aae

Comment (mulengawilfred) [reply web/e2e/README.md:58]: Thanks. Looks good.

**Review by airslice — APPROVED** [review]

**Pull request merged**